smwwims.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@smwwims"
+project: Autonomous Plant Farm
+---
+
+# Autonomous Plant Farm
+
+## Summary
+
+I have always had a keen interest in botany, but unfortunately my gardening skills do not quite match my programming skills. 
+That is why I came up with the idea of building a fully automatic cultivation machine. In contrast to the "Smart Planting Sets" 
+available on the market, my machine is not a simple control circuit, but a regulation circuit. This means that the water and light 
+supply is not simply controlled by the time but responds to the needs of the plants via sensors.
+
+## Plan
+
+1. Preparing the plant pot:   Install the water tank, demarcate the electronic part
+2. Water supply:              Install pump, install water pipe
+3. Sensors:                   Wire the sensors
+4. Light:                     Install and wire the plant light
+5. Programming:               Make it all work through software
+
+## Budget
```

Comment:
does your budget include taxes and shipping? if it doesn't, could you please include it?

Reply:
Yes, the budget includes taxes and shipping. All items are from Amazon Austria, the taxes and shipping costs are already included in the prize.